packages/discord.js/src/client/websocket/WebSocketManager.js

```diff
@@ -136,7 +136,7 @@ class WebSocketManager extends EventEmitter {
       shards: recommendedShards,
       session_start_limit: sessionStartLimit,
     } = await this.client.rest.get(Routes.gatewayBot()).catch(error => {
-      throw error.httpStatus === 401 ? invalidToken : error;
+      throw error.status === 401 ? invalidToken : error;
```

Comment:
Out of scope

Reply:
I believe this was recently merged so you might just need to rebase